app/vmctl/vm_native.go

```diff
@@ -23,6 +26,55 @@ type vmNativeProcessor struct {
 	dst          *vmNativeClient
 	src          *vmNativeClient
 	interCluster bool
+	backoff      *backoff.Backoff
+
+	s             *stats
+	cc            int
+	requestsLimit int
+}
+
+type stats struct {
+	sync.Mutex
+	bytes        uint64
+	requests     uint64
+	retries      uint64
+	startTime    time.Time
+	idleDuration time.Duration
```

Comment:
idleDuration is unused. That's why it is equal to 0 in the readme example

Reply:
removed this field